src/components/app/app.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { fireEvent, render, screen, within } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { App } from './app'
+import React from 'react'
+import * as deck from '../../services/deck'
+import { CardSuit, CardValue } from '../../interfaces/card.interface'
+
+test('Renders app', () => {
+    render(<App />)
+
+    expect(screen.getByText('DEALER')).toBeInTheDocument
+    expect(screen.getByText('PLAYER')).toBeInTheDocument
+})
+
+test('Dealer is dealt a seven and ten', () => {
+    jest.spyOn(deck, 'dealHand').mockReturnValue({
```

Comment:
we are going to be mocking the dealHand function **a lot**. Thus, let's make a helper method at bottom of this file to reduce the amount of code needed in each test to setup the hand to be dealt.

something like this
```
mockDealHand(firstCardValue: CardValue, secondCardValue: CardValue, finished: boolean = true, firstCardSuit: CardSuit = CardSuit.CLUBS, secondCardSuit: CardSuit.DIAMONDS)
```

the suit really does not affect behavior so we put those ones last to provide default values. makes signature easier on caller as they only need to provide card values